src/FSLibrary/BenchmarkDaemonSet.fs

```diff
@@ -0,0 +1,280 @@
+// Copyright 2025 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module BenchmarkDaemonSet
+
+open k8s
+open k8s.Models
+open System
+open System.Collections.Generic
+open Logging
+open StellarKubeSpecs
+open StellarNetworkCfg
+open StellarNetworkDelays
+open StellarCoreCfg
+
+// Docker image for iperf3 - a tool for measuring network throughput and latency
+// We use it to run bidirectional P2P network performance tests between stellar-core nodes
+let benchmarkImage = "networkstatic/iperf3:latest"
+
+let benchmarkLabels = Map.ofSeq [ "app", "network-benchmark" ]
+
+// Extract short name from stellar-core DNS patterns for use in benchmark pod names
+// We keep the full name including "node-" prefix to ensure DNS resolution works
+// Examples: "node-0-0" -> "node-0-0", "sdf-0" -> "sdf-0", "pn-0" -> "pn-0"
+let extractShortNameFromStellarCoreDns (fullName: string) : string =
+    // Keep the full name to ensure DNS resolution works correctly
+    // The benchmark pods use the same naming pattern as stellar-core pods
+    fullName
```

Comment:
I don't understand, it looks like this is just the identity function (returns `fullName`) and doesn't extract anything. Do we even need this function?